package.json

```diff
@@ -112,21 +112,21 @@
     "express-logging": "1.1.1",
     "extract-zip": "2.0.1",
     "fastest-levenshtein": "1.0.16",
-    "fastify": "4.28.1",
+    "fastify": "4.29.0",
     "find-up": "7.0.0",
     "flush-write-stream": "2.0.0",
     "folder-walker": "3.2.0",
     "from2-array": "0.0.4",
     "fuzzy": "0.1.3",
-    "get-port": "5.1.1",
+    "get-port": "7.1.0",
```

Comment:
now requires node 18 (we require 18) and uses esm, plus irrelevant changes: https://github.com/sindresorhus/get-port/releases